.github/workflows/node.js.yml

```
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x, stable, lts]
```

```suggestion
        node-version: [10.x, 12.x, 14.x, 15.x]
```

Using `stable` and `lts` in `node-version` array is not supported, so this suggestion removes those items.

Valid syntax:

- `12.x`
- `10.15.1`
-  `>=10.15.0`